compute_transfer_examples/README.md

```diff
@@ -0,0 +1,152 @@
+# Globus Compute and Transfer Flow Examples
+
+This guide demonstrates how to build flows that combine Globus Compute and Globus Transfer to process and move data. You'll learn how to create two different flows that archive (tar) files and transfer them from a source collection to a destination collection.
+
+## Prerequisites
+
+Before starting, ensure you have a shared GCS collection and Globus Compute endpoint.
+If you haven't set these up, follow [this guide](https://docs.globus.org/globus-connect-server/v5.4/) for setting up the GCS collection, and [this guide](https://globus-compute.readthedocs.io/en/latest/endpoints/installation.html) for setting up the Globus Compute endpoint. **Note**: The GCS collection and Globus Compute endpoint must have access to the same filesystem at the same path.
+
+## Register the Globus Compute Function
+
+First, register the `do_tar` Compute function that your flows will invoke to create the output tarfiles. Run the provided python script:
+
+```bash
+./transfer_compute_example/register_compute_func.py
+```
+
+and copy down the Compute function's UUID.
+
+**Important**: Use the same Python version for registration as the one running on your Globus Compute Endpoint.
+
+### The `do_tar` Compute function
+
+`do_tar` takes four parameters that the flow will need to provide:
+
+| Parameter | Description |
+|-----------|-------------|
+| `src_paths` | String or list of path(s) to files/directories to archive |
+| `dest_path` | Where to write the tar.gz archive (directory or file path) |
+| `transform_from` | The path prefix to replace (default: "/") |
+| `transform_to` | The prefix to use for absolute paths (default: "/") |
+
+### Path Transformation Explained
+
+The path transformation parameters `transform_from` and `transform_to` handle differences between how paths appear in the Globus ecosystem versus how they exist on the actual GCS collection's filesystem.
```

Comment:
I'm not quite sure I follow this as intended, so I'm not quite sure if this revision is correct, but my guess is that we mean to say:

```suggestion
The parameters `transform_from` and `transform_to` handle differences between paths as exposed by the GCS collection and paths on the underlying filesystem.
```